.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -92,8 +92,7 @@ swap:
   componentization: component-based development|component model|component architecture|shared components
   componentize: develop components
   comprised of: consist of
-  computer farm|computer ranch: server cluster
-  computer farm|computer ranch: server farm
+  computer farm|computer ranch: server cluster|serverfarm
```

Comment:
s/serverfarm/server farm

Reply:
sorry, I got rid of the duplicate and made a new error while at it